src/hotspot/cpu/ppc/assembler_ppc.inline.hpp

```diff
@@ -338,28 +338,34 @@ inline void Assembler::insrwi(  Register a, Register s, int n,   int b)
 
 // PPC 1, section 3.3.2 Fixed-Point Load Instructions
 inline void Assembler::lwzx( Register d, Register s1, Register s2) { emit_int32(LWZX_OPCODE | rt(d) | ra0mem(s1) | rb(s2));}
+inline void Assembler::lwz(  Register d, Address &a) { lwz(d, a.disp(), a.base()); }
```

Comment:
The load instructions need the same adaptation as the store instructions (just without the tmp Register).
```
inline void Assembler::lwz(  Register d, Address &a) {
  lwz(d, a.index() != noreg ? RegisterOrConstant(a.index()) : RegisterOrConstant(a.disp()), a.base());
}
```